clickhouse_search/models.py

```diff
@@ -645,6 +647,7 @@ class Meta(BaseEntries.Meta):
         )
 
 class EntriesSv(BaseEntries):
+    MAX_XPOS_FILTER_INTERVALS = 0
```

Comment:
for SNV_INDELs for small gene lists its more efficient to just use an xpos range instead of the bitmap filter. For SVs, since we essentially can't use the xpos filter at all its always more effective to use the bitmap filter